app/gui/integration-test/dashboard/actions/DrivePageActions.ts

```diff
@@ -229,6 +270,28 @@ export default class DrivePageActions<Context> extends PageActions<Context> {
             }),
         )
       },
+      expandDirectory(index: number) {
+        return self.step(`Expand drive table row #${index}`, async (page) => {
+          const expandButton = locateAssetRows(page)
+            .nth(index)
+            .getByTestId('directory-row-expand-button')
+
+          await expect(expandButton).toHaveAttribute('aria-label', TEXT.expand)
+
+          return expandButton.click()
```

Comment:
nit: i prefer `await`ing all promises even if the result is the same

Reply:
Even though _techically_ there's no difference between `return` and `await`, we throw the return of the callback anyway, so yeah, await here is preferable.